content/docs/manage/api-keys.md

```diff
@@ -41,16 +41,17 @@ To revoke an API key:
 
 The following `cURL` example uses the `/projects` endpoint to retrieve projects that belong to your Neon account.
 
-```bash
-curl -X GET -H "Authorization: Bearer $NEON_API_KEY" "accept: application/json"
-"https://neon.tech/api-reference/v2" | jq
+```curl
+curl 'https://console.neon.tech/api/v2/projects' \
+  -H 'accept: application/json' \
```

Comment:
```suggestion
  -H 'Accept: application/json' \
```

Headers are case-insensitive, but I think it looks neat when they are capitalised consistently.